### PR TITLE
custom folder for CMC & CMC Legal

### PR DESCRIPTION
**Pull request by ManishParyani**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-9185
https://tools.hmcts.net/jira/browse/ROC-9186

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: cmc/index.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            </span>
+          </a>
+        </div>
+
+        <div class="govuk-header__content">
+          <a
+            href="/"
+            class="govuk-header__link govuk-header__link--service-name"
+          >
+            <!-- TODO Add your service name here -->
```
Comment (timja) [inline cmc/index.html:115]: should add your Service name here which matches your service in live

File: cmclegal/index.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+            </span>
+          </a>
+        </div>
+
+        <div class="govuk-header__content">
+          <a
+            href="/"
+            class="govuk-header__link govuk-header__link--service-name"
+          >
+            <!-- TODO Add your service name here -->
```
Comment (timja) [inline cmclegal/index.html:115]: and here

File: cmc/index.html (hunk 1 above)
Comment (ManishParyani) [reply cmc/index.html]: @timja  Thanks for reviewing this PR, I have updated the service name.

File: cmclegal/index.html (hunk 2 above)
Comment (ManishParyani) [reply cmclegal/index.html]: @timja  Thanks for reviewing this PR, updated.

**Review by timja — APPROVED** [review]

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**